packages/match-list/src/answer-list.jsx

```diff
@@ -12,12 +12,19 @@ export class AnswerList extends React.Component {
     disabled: PropTypes.bool.isRequired,
     onSessionChange: PropTypes.func,
     onRemoveAnswer: PropTypes.func,
-    placeAnswer: PropTypes.func.isRequired,
+    onPlaceAnswer: PropTypes.func.isRequired,
     instanceId: PropTypes.string.isRequired,
     model: PropTypes.object.isRequired,
     prompt: PropTypes.string
   };
 
+  constructor(props) {
+    super(props);
+
+    this.getAnswerFromSession = this.getAnswerFromSession.bind(this);
```

Comment:
i think you can remove these and as part of the class definition use arrow functions: 
```javascript
class Foo {
  myBoundFn = () => {
  }
}
```